projects/plugins/jetpack/extensions/blocks/ai-image/edit.js

```diff
@@ -14,6 +14,31 @@ import { useSelect, useDispatch } from '@wordpress/data';
 import { useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 
+function getRandomItem( arr ) {
+	// get random index value
+	const randomIndex = Math.floor( Math.random() * arr.length );
+	return arr[ randomIndex ];
+}
+
+const PLACEHOLDER = getRandomItem( [
+	'earth reviving after human extinction, a new beginning, nature taking over buildings, animal kingdom, harmony, peace, earth balanced --version 3 --s 1250 --uplight --ar 4:3 --no text, blur',
```

Comment:
Do we want those to be translated?

Reply:
Frankly no, I don't think so. While I expect them to work in other languages, the translators would have to also check if these are producing decent results as they would most likely be differnt. Let's have them separate for now